ext/pdo/pdo_dbh.c

```diff
@@ -894,18 +893,13 @@ PHP_METHOD(PDO, getAttribute)
 		RETURN_FALSE;
 	}
 
-	switch (dbh->methods->get_attribute(dbh, attr, return_value)) {
-		case -1:
-			PDO_HANDLE_DBH_ERR();
-			RETURN_FALSE;
-
-		case 0:
-			pdo_raise_impl_error(dbh, NULL, "IM001", "driver does not support that attribute");
-			RETURN_FALSE;
-
-		default:
-			return;
+	if (dbh->methods->get_attribute(dbh, attr, return_value) == false) {
```

Comment:
This was previously a 3-way return value (error, not supported, success). You have dropped support for the error case.

Reply:
I haven't seen it used in our DB drivers, and didn't really understand what the code flow was, but I can revert this change.

Reply:
Probably none of our DB drivers can error while getting an attribute (apart from not supporting it completely), but I think from an API perspective this should remain supported.

Reply:
ACK